src/utilities.cpp

```diff
@@ -53,4 +55,41 @@ std::string convertDoubleToString(double value)
     return strs.str();
 }
 
+bool isCellMLIntegerCharacter(char c) {
+    const std::set<char> validIntegerCharacters = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'};
+    return validIntegerCharacters.find(c) != validIntegerCharacters.end();
+}
+
+bool convertToInt(const std::string &candidate, int *value)
```

Comment:
I believe we are using C++11? If so, we might want to use [`stoi()`](http://www.cplusplus.com/reference/string/stoi/).

Reply:
We are using it, we are just wrapping over it so we can do some manipulation checking if required.

Reply:
Oops, my bad, I didn't spot it!